Skripte/skript.txt

```diff
@@ -1554,6 +1623,59 @@ var DefaultInhalte = {
         },
 };
 
+// device_name wird aus dem <id.common.name> geholt
+// falls er dort nicht steht, dann kann hier eine eigene Funktion zum ermitteln erstellt werden
+function getDevName(id) {

```

Comment:
Kann diese Funktion mit getCondName zusammengeführt werden? Ist aktuell 2-mal der selbe Code oder übersehe ich da etwas?
Möglicher Optimierunspotenzial: Wenn der reale Name in einem anderen State steht, was steht dann unter "Name" im Level-State? Beispiel:
`adaptername.0.abc.rolladen.level ` hat unter Namen `abc.rolladen.level`, dann könnte dieser Vergleich durchgeführt werden. Wenn Name mit State-ID mit den letzten drei Elementen (getrennte Elemente mit ".") übereinstimmt, dann nimm Name aus `adaptername.0.abc.rolladen.name`. Die alternative Endung, hier `name`, könnte oben im Bereich der Variablen angegeben werden. Dafür könnten sowohl die Angaben für `getDeviceNameMode` und `getConditionNameMode` im oberen Var-Bereich entfallen.